org.sonatype.m2e.webby/src/org/sonatype/m2e/webby/internal/config/WarConfigurationExtractor.java

```diff
@@ -198,8 +198,7 @@ public WarConfiguration getConfiguration(IMavenProjectFacade mvnFacade, MavenPro
   private String resolve(String basedir, String path) {
     String result = path;
     if(path != null && basedir != null) {
-      path = path.replace('\\', File.separatorChar).replace('/', File.separatorChar);
-
+      path = path.replace('\\', '/');
```

Comment:
In fact the old code don't work in windows with tomee in war extra classpath, because we have some "\" character
I think "/" it's better to work with every OS
=> I Have test this on Windows and Mac Os X